libraries/linux_module.rb

```diff
@@ -60,7 +60,11 @@ def version
   def command
     # Lets just ensure the last line in the kernel module's configuration is 'install /bin/true'
     # this is enough to be sure the module will not be loaded on next reboot or run of modprobe
-    modinfo_cmd = "/sbin/modprobe -n -v #{@module} | tail -n 1 | awk '{$1=$1;print}'"
+    modinfo_cmd = if inspec.os.redhat? || inspec.os.name == 'fedora'
+                    "/sbin/modprobe -n -v #{@module} | tail -n 1 | awk '{$1=$1;print}'"
+                  else
```

Comment:
With this change we expect all linux other then redhat to behave like debian. At this point, it feels safer to do the if check for debian and use the redhat one as default. What do you think?